config/locales/de.yml

```diff
@@ -355,75 +355,75 @@ de:
+      p2: Einige Teile der codebar Website funktionieren auch ohne Cookies, und Sie können generell wählen, ob Sie Cookies akzeptieren wollen. Die meisten Webbrowser sind so eingestellt, dass sie Cookies standardmäßig akzeptieren. Sie können Cookies jedoch auch selbst über den Cookie-Manager Ihres Browsers löschen. Befolgen Sie dazu bitte die Anweisungen Ihres Webbrowsers. Bitte beachten Sie, dass die Deaktivierung von Cookies Ihre Sitzung zurücksetzt, die automatische Anmeldung deaktiviert und die Verfügbarkeit und Funktionalität unserer Website sowie die Dienstleistungen, die wir Ihnen anbieten können, beeinträchtigen kann.
       sub:
-        title: Below are the cookies used by the codebar website.
+        title: Im Folgenden werden die von der codebar Website verwendeten Cookies aufgeführt.
       table:
         header:
           h1: Name
-          h2: Purpose
-          h3: Expires
+          h2: Zweck
+          h3: Abgelaufen
       functional:
         title: Functional Cookies
-        description: Stores information about your session.
+        description: Speichert Informationen über Ihre Sitzung.
         planner_name: _planner_session
-        planner_purpose: Stores session data
-        planner_expiry: Expires after 24 hours
+        planner_purpose: Speichert Sitzungsdaten
+        planner_expiry: Läuft nach 24 Stunden ab
       google_analytics:
         title: Google Analytics
-        description: Google Analytics collects information about how you use the codebar website. This enables us to identify how busy the website is and possible improvements.
-        info: We don’t allow Google to use or share our analytics data.
-        opt_out: You can [opt out of Google Analytics](https://tools.google.com/dlpage/gaoptout).
+        description: Google Analytics sammelt Informationen darüber, wie Sie die codebar Website nutzen. So können wir feststellen, wie stark die Website genutzt wird und mögliche Verbesserungen vornehmen.
+        info: Wir erlauben Google nicht, unsere Analysedaten zu verwenden oder weiterzugeben.
+        opt_out: Sie können [Google Analytics deaktivieren](https://tools.google.com/dlpage/gaoptout).
         ga_name: _ga
-        ga_purpose: Tracks if you have visited codebar.io before
-        ga_expiry: After 2 years
+        ga_purpose: Spuren, wenn Sie codebar.io bereits besucht haben
```

Comment:
`Spuren` is not really right, I think you would use `Track` as well here in German.